pkg/k8shandler/desirednodestate.go

```diff
@@ -207,6 +210,35 @@ func (cfg *desiredNodeState) CreateOrUpdateNode(owner metav1.OwnerReference) err
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("Could not create node resource: %v", err)
 		}
+
+		// set it back to being paused
+		cfg.Paused = true
+		nretries := -1
+		if getErr := node.query(); getErr != nil {
```

Comment:
I don't think you need to do the `node.query()` here if you are doing it in the retry loop

Reply:
good point, i'll remove it